### PR TITLE
dlgView

### PR DESCRIPTION
**Pull request by lilyclements**
@rdstern - after yesterdays conversation:
Added an Option to View All - this is always going to occur if you select to view in a separate window, and is optional if you select to view in output window (default to specify interval)
I could not give an option (yet?) to give an interval for rows to view, however.

### CODE REVIEW

**Comment by lilyclements** [comment]
Also, if you try to run this there is an error.
This is fixed in #2379 by Francois. It is waiting to be merged in after a few new changes.

File: instat/dlgView.vb [hunk 1]
```diff
@@ -95,18 +99,20 @@ Public Class dlgView
...
         If rdoDispSepOutputWindow.Checked Then
-            XandNParameters()
-            If rdoBottom.Checked Then
-                clsView.AddParameter("x", clsRFunctionParameter:=clsTail)
-            ElseIf rdoTop.Checked Then
-                clsView.AddParameter("x", clsRFunctionParameter:=clsHead)
-            End If
             ucrBase.clsRsyntax.SetBaseRFunction(clsView)
-            ucrBase.clsRsyntax.AddParameter("title", Chr(34) & ucrSelectorForView.strCurrentDataFrame & Chr(34))
+            ucrBase.clsRsyntax.AddParameter("", ucrSelectorForView.strCurrentDataFrame)
```
Comment (dannyparsons) [inline instat/dlgView.vb]: Every parameter should have a name. Is this for the "View" function?

Comment (lilyclements) [inline instat/dlgView.vb]: Yes, I'm trying to just do:
View(Dataframe)

Comment (dannyparsons) [inline instat/dlgView.vb]: Then the parameter name is `x`.

Comment (lilyclements) [inline instat/dlgView.vb]: Great, added it in. Sorry!

**Pull request merged**